s3/src/main/scala/monix/connect/s3/S3.scala

```diff
@@ -885,6 +884,111 @@ trait S3 { self =>
     } yield s3Object
   }
 
+  /**
+    * Returns the most recently uploaded object in a bucket.
+    *
+    * ==Example==
+    *
+    * {{{
+    * import cats.effect.Resource
+    * import monix.connect.s3.S3
+    * import monix.eval.Task
+    * import software.amazon.awssdk.auth.credentials.{AwsBasicCredentials, StaticCredentialsProvider}
+    * import software.amazon.awssdk.regions.Region
+    * import monix.execution.Scheduler.Implicits.global
+    *
+    * val bucket = "my-bucket"
+    * val prefix = "prefix/to/list/keys"
+    * val s3AccessKey: String = "TESTKEY"
+    * val s3SecretKey: String = "TESTSECRET"
+    * val basicAWSCredentials = AwsBasicCredentials.create(s3AccessKey, s3SecretKey)
+    * val staticCredProvider = StaticCredentialsProvider.create(basicAWSCredentials)
+    *
+    * val s3: Resource[Task, S3] = S3.create(staticCredProvider, Region.AWS_GLOBAL)
+    *
+    * s3.use{s3 => s3.listLatestObject(bucket, prefix = Some(prefix)).toListL}.runToFuture
+    * }}}
+    *
+    * @note                To use this operation in an AWS (IAM) policy, you must have permissions to perform
+    *                      the `ListBucket` action. The bucket owner has this permission by default and can grant it.
+    * @param bucket        target S3 bucket name of the object to be downloaded.
+    * @param prefix        limits the response to keys that begin with the specified prefix.
+    * @param requestPayer  confirms that the requester knows that she or he will be charged for
+    *                      the list objects request in V2 style.
+    *                      Bucket owners need not specify this parameter in their requests.
+    * @return an [[Observable]] that emits the [[S3Object]]s.
+    */
+
+  def listLatestObject(
```

Comment:
why not `listLatestNObjects` too?

Reply:
I should probably also add listOldestObject as well so we have listLastestNObject, ListLastestObject, listOldestNObjects, and listOldestObject. Thanks for being patient with me and reviewing all my code!

Reply:
no problem :) 
okay yeah that should not be a big deal!
you could just implement `listLastestObject`from `listLastestNObject`, and it could just return `Task` instead of `Observable`
```
def listLatestObject(bucket, path): Task[Option[S3Object]] = {
 listLatestNObjects(bucket, path, 1).head //it returns none if there is no s3object present under that prefix
}
```